frontend/src/helpers/serverError.js

```diff
@@ -12,27 +14,50 @@ const serverErrorToString = (error) => {
   return error.message
 }
 
+function getApiTranslation(locale, violation) {
+  if (!locale) {
+    return null
+  }
+  const matchingTranslation = [locale, ...fallbackLocalesFor(locale)]
+    .map((locale) => violation.i18n?.translations[locale])
```

Comment:
If `i18n` is `undefined`, then `i18n?.translations` is silently `undefined` as well, and the array access will throw an error. You can use the optional chaining array access operator:
```suggestion
    .map((locale) => violation.i18n?.translations?.[locale])
```
Looks strange, but works.

Reply:
It's our api, so i18n always has this key, even when the env variable TRANSLATE_ERRORS_TO_LOCALES is empty.
Will fix

Reply:
done